boolean/boolean.py

```diff
@@ -437,9 +426,10 @@ def tokenize(self, expr):
                         break
                 position -= 1
 
-            try:
-                yield TOKENS[tok.lower()], tok, position
-            except KeyError:
+            value = TOKENS.get(tok.lower())
+            if value is not None:
```

Comment:
You can write: `if value:` instead? is there cases where we we get an empty token otherwise?